Xamarin.Forms.Core/OnIdiom.cs

```diff
@@ -10,6 +10,8 @@ public class OnIdiom<T>
 
 		public T TV { get; set; }
 
+		public T Watch { get; set; }
```

Comment:
adding an enum value is potentially a breaking change

Reply:
This should actually be switched to a string like the Platform was

Reply:
@mattleibow I see what you're doing. planning to have a Forms app running in your Tizen microwave!